.github/workflows/Release.yml

```diff
@@ -6,13 +6,31 @@ on:
       - created
 
 jobs:
-  build:
+  test:
     runs-on: "ubuntu-20.04"
     steps:
       - uses: actions/checkout@v2
       - name: Run tests
         run: cargo test --verbose
-      - name: Release on Crates.io
+
+  publish-dgc:
+    runs-on: "ubuntu-20.04"
+    needs: test
+    if: startsWith(github.ref, 'refs/tags/dgc/')
```

Comment:
Isn't this going to match both `dgc` and `dgc-italy-core`?

Is this intended (one tag -> multiple crates released) or not?

Reply:
No, in theory (and according to the tests I made in my fork) it should only work if the tag starts with `dgc/` (because of the trailing slash).

Reply:
Makes sense, thanks for double checking